app/views/planning_application/withdraw_or_cancels/show.html.erb

```diff
@@ -0,0 +1,97 @@
+<% content_for :page_title do %>
+  Withdraw or cancel application - <%= t('page_title') %>
+<% end %>
+
+<% add_parent_breadcrumb_link "Home", planning_applications_path %>
+<% add_parent_breadcrumb_link "Application", planning_application_path(@planning_application) %>
+
+<% content_for :title, "Withdraw or cancel application" %>
+
+<div class="govuk-grid-row"></div>
+  <h1 class="govuk-heading-l">Withdraw or cancel application</h2>
+
+  <%= render "shared/planning_application_address_and_reference" %>
```

Comment:
In this [pr](https://github.com/unboxed/bops/pull/918) I am going to delete the partial `shared/planning_application_address_and_reference`, as we are going to follow this format
```
<%= render(
  partial: "shared/proposal_header",
  locals: { heading: "HEADING"  }
) %> 
```
and then will be the page grid row.

Can you update it 🙏 ?